pkg/dogstatsd/server_test.go

```diff
@@ -386,10 +403,13 @@ func TestExtraTags(t *testing.T) {
 }
 
 func TestDebugStats(t *testing.T) {
-	metricOut := make(chan []metrics.MetricSample)
-	eventOut := make(chan []*metrics.Event)
-	serviceOut := make(chan []*metrics.ServiceCheck)
-	s, err := NewServer(metricOut, eventOut, serviceOut)
+	agg := aggregator.NewBufferedAggregator(
```

Comment:
What do you think about using `mockAggregator()` instead?
This comment could be applied to the whole file.